src/executor/query/QueryStorageExecutor.h

```diff
@@ -4,46 +4,42 @@
  * attached with Common Clause Condition 1.0, found in the LICENSES directory.
  */
 
-#ifndef EXECUTOR_MUTATE_MUTATEEXECUTOR_H_
-#define EXECUTOR_MUTATE_MUTATEEXECUTOR_H_
-
-#include "common/interface/gen-cpp2/storage_types.h"
-#include "common/clients/storage/GraphStorageClient.h"
+#ifndef EXECUTOR_QUERYSTORAGEEXECUTOR_H_
+#define EXECUTOR_QUERYSTORAGEEXECUTOR_H_
 
 #include "executor/Executor.h"
+#include "common/clients/storage/StorageClientBase.h"
 
 namespace nebula {
 namespace graph {
 
-class MutateExecutor : public Executor {
-public:
-    MutateExecutor(const std::string &name, const PlanNode *node, QueryContext *qctx)
+// It's used for data write/update/query
```

Comment:
It's better to move this file to `executor/` rather than `executor/query` since it's used by mutate/query executors.

Reply:
Fixed, thanks.